filecoin-proofs/examples/zigzag.rs

```diff
@@ -214,21 +253,13 @@ fn do_the_work<H: 'static>(
     let vanilla_proving = start.elapsed();
     total_proving += vanilla_proving;
 
-    let proving_avg = total_proving;
-    let proving_avg =
-        f64::from(proving_avg.subsec_nanos()) / 1_000_000_000f64 + (proving_avg.as_secs() as f64);
-
-    // -- print statistics
-
-    //    let serialized_proofs = proofs.iter().fold(Vec::new(), |mut acc, p| {
-    //        acc.extend(p.serialize());
-    //        acc
-    //    });
-    //    let avg_proof_size = serialized_proofs.len() / samples as usize;
-    //
-    //info!(target: "stats", "Average proof size {}", prettyb(avg_proof_size));
+    let serialized_proofs = (serde_json::to_string(&all_partition_proofs))
```

Comment:
there is to_bytes method on serde json, also you can create a streaming serializer with serde json, removing the need to buffer all into memory but rather write them out immediately